app/serializers/search/postgres_user_serializer.rb

```diff
@@ -0,0 +1,17 @@
+module Search
+  # to be renamed when ES is gone
+  class PostgresUserSerializer < ApplicationSerializer
+    attribute :class_name, -> { "User" }
+    attributes :id, :path
+    attribute :title, &:name
```

Comment:
that's how the frontend expects the data, we can't change it now, until we can refactor both backend and frontend, post ES-removal